tests/load/c/ginkgo_test.go

```diff
@@ -64,30 +64,36 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 var _ = ginkgo.Describe("[Load Simulator]", ginkgo.Ordered, func() {
 	var (
-		network  *tmpnet.Network
-		tracker  *load.Tracker[common.Hash]
-		logger   = logging.NewLogger("c-chain-load-testing", logging.NewWrappedCore(logging.Info, os.Stdout, logging.Auto.ConsoleEncoder()))
-		tc       = e2e.NewTestContext()
-		r        = require.New(tc)
-		registry = prometheus.NewRegistry()
-	)
+		network *tmpnet.Network
+		tc      *e2e.GinkgoTestContext
+		tracker *load.Tracker[common.Hash]
 
-	tracker, err := load.NewTracker[common.Hash](registry)
-	r.NoError(err)
+		logger = logging.NewLogger("c-chain-load-testing", logging.NewWrappedCore(logging.Info, os.Stdout, logging.Auto.ConsoleEncoder()))
+	)
 
 	ginkgo.BeforeAll(func() {
-		tc := e2e.NewTestContext()
-		r := require.New(tc)
-		env := e2e.GetEnv(tc)
+		var (
+			tc  = e2e.NewTestContext()
+			env = e2e.GetEnv(tc)
+		)
```

Comment:
nit a bit unneeded diffs; fine if this is new code though. I'm a bit of a minimal-git-diffs maniac please forgive me 😄 (for somehow decent reasons though)

```suggestion
		tc := e2e.NewTestContext()
		env := e2e.GetEnv(tc)
```